### PR TITLE
Restructure server folders

### PR DESCRIPTION
**Pull request by prakhar1912**
I have restructured the server folder to make it more streamlined. Now, the structure is as follows:

new folder structure in server

### CODE REVIEW

**Comment by niinpatel** [comment]
Looks good to me. Assign it to nimish too and then merge when approved.

**Comment by nimish-gupta** [comment]
@prakhar1912 meanwhile add babel support in this so that you can use import/export or latest es features which are missing in latest NODE stable version. Otherwise LGTM.

File: server/package.json [hunk 1]
```diff
@@ -4,14 +4,17 @@
   "description": "Backend for rattle-battle",
   "main": "index.js",
   "scripts": {
-    "start": "node index.js",
-    "lint": "npx eslint index.js",
+    "start": "node src/index.js",
+    "start:dev": "nodemon src",
+    "lint": "npx eslint src/index.js",
```
Comment (niinpatel) [inline server/package.json]: I don't think run eslint on all files not just index.js

**Comment by prakhar1912** [comment]
> @prakhar1912 meanwhile add babel support in this so that you can use import/export or latest es features which are missing in latest NODE stable version. Otherwise LGTM.

@nimish-gupta Okay, I will merge this PR, and make a new one with Babel support.

**Pull request merged**